packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ConfigureModelResponseHandler.java

```diff
@@ -89,6 +84,10 @@ public void onSuccess(TaskResponse resp) {
                     });
                 });
 
+                if(modelCopy.isRegnet()) {
+                    modelCopy.getModel().put("parameters", objectMapper.convertValue(modelParameters, JsonNode.class));
+                }
+
```

Comment:
for regnet, since parameters are not defined on the object we do not get a reference to the original object, we need to set it